### DISCUSSION TITLE
Can't use nginx to proxy the postal web

### DISCUSSION THREAD

**Discussion by saobilin** [postalserver/postal#1827] [Help with using Postal]
## Describe the bug

When i stop the caddy docker and run the nginx,the postal web can't be login.
It report this :
The change you wanted was rejected.
Maybe you tried to change something you didn't have access to.

If you are the application owner check the logs for more information.

## To Reproduce

1. Go to stop the caddy docker
2. Run the nginx,the nginx config file is ok and have web ssl config
The nginx config is :
server
{
    listen 80;
        listen 443 ssl http2;
    server_name ****;
    index index.php index.html index.htm default.php default.htm default.html;
   
    if ($server_port !~ 443){
        rewrite ^(/.*)$ https://$host$1 permanent;
    }
    #HTTP_TO_HTTPS_END
    ssl_certificate    /root/ssl/***.icu.crt;
    ssl_certificate_key    /root/ssl/***.icu.key;
    ssl_protocols TLSv1.1 TLSv1.2 TLSv1.3;
    ssl_ciphers EECDH+CHACHA20:EECDH+CHACHA20-draft:EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:!MD5;
    ssl_prefer_server_ciphers on;
    ssl_session_cache shared:SSL:10m;
    ssl_session_timeout 10m;
    add_header Strict-Transport-Security "max-age=31536000";
    error_page 497  https://$host$request_uri;
location /
{
    proxy_pass http://127.0.0.1:5000;
    proxy_set_header Host $host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header REMOTE-HOST $remote_addr;
}
}
3. this time when you visit the postal web proxy by nginx,you can visit the login Interface,But when you input your email address and password,it show the error.
The change you wanted was rejected.
Maybe you tried to change something you didn't have access to.

If you are the application owner check the logs for more information.

## Expected behaviour

Use nginx to replace the caddy

## Environment details 

 - OS: Windows
 - Browser Chrome
 - Version 
 - Type Desktop

## Additional information/context

Add any other context about the problem here. It is particularily useful to include log extracts (after removing private information).

**Comment by willpower232** [comment]
Seems a few people are struggling with their proxy settings in #1781

**Comment by 0xjas0** [comment]
I was having the issue and figured that it was due to csrf verification, with proxy_pass the host was getting rewritten and that can be solved using `proxy_set_header Host $http_host;` under proxy_pass
